catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -154,13 +166,44 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       query.setFacetMinCount(textFacetProp.getMinFacetCount());
     }
 
-    long totalHits;
-    List<Result> results = new ArrayList<>();
-    Map<String, Serializable> responseProps = new HashMap<>();
+    Serializable suggestQueryProp = request.getPropertyValue(SUGGESTION_QUERY_KEY);
+    Serializable suggestContextProp = request.getPropertyValue(SUGGESTION_CONTEXT_KEY);
+    Serializable suggestDictProp = request.getPropertyValue(SUGGESTION_DICT_KEY);
+
+    if (suggestQueryProp instanceof String && suggestContextProp instanceof String) {
+      query = new SolrQuery();
+      query.setRequestHandler("/suggest");
+      query.setParam(SUGGEST_Q, (String) suggestQueryProp);
+      query.setParam(SUGGEST_CONTEXT_FILTER_QUERY, (String) suggestContextProp);
+      query.setParam(SUGGEST_DICT, (String) suggestDictProp);
```

Comment:
❓ Is it ok if the `suggestDictProp` is `null`?

Reply:
SolrJ appears to use a LinkedHashMap under the hood for params - this should handle a null key gracefully and have the effect of leaving the parameter out - effectively a no-op.

Reply:
This will also cause Solr to use a suggester named `default` to look for suggestions.